recipes/perl-capture-tiny/0.34/build.sh

```diff
@@ -1,25 +0,0 @@
-#!/bin/bash
-
-# If it has Build.PL use that, otherwise use Makefile.PL
```

Comment:
Why do you remove some perl packages? Was this a bad merge, rebase? Can you please rebase against master and squash your commits?